src/main/java/com/jfrog/ide/idea/scan/ScanManagersFactory.java

```diff
@@ -121,7 +101,6 @@ public void refreshScanManagers() throws IOException {
             scanManagers.put(projectHash, scanManager);
         } else {
             createScanManagerIfApplicable(scanManagers, projectHash, ScanManagerTypes.MAVEN, "");
```

Comment:
Let's add a description here, explaining why the maven scan manager is created differently.